@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/fn_loadoutImportJson.sqf

```diff
@@ -0,0 +1,91 @@
+/* ----------------------------------------------------------------------------
+Function: fza_fnc_loadoutImportJson
+
+Description:
+	Import of website heicopter configuration and apply it to the helicopter
+	
+Parameters:
+	_heli - The helicopter to get information from [Unit].
+
+Returns:
+
+Examples:
+	[vehicle player, Json Data] call fza_loadoutImportJson
+
+Author:
+	Rosd6(Dryden)
+---------------------------------------------------------------------------- */
+params ["_heli","_json"];
+if (_json == "") exitwith {};
+private _settings = [_json] call CBA_fnc_parseJSON;
+private _PylonArraycheck  = ["pylon1","pylon2","pylon3","pylon4"];
+private _pylonRktCheck = ["zoneA","zoneB","zoneE"];
+private _pylonMslCheck = ["ul","ur","ll","lr"];
+private _MagazineArray = [];
+private _MagazineIndex = 16;
+
+//System Settings
+private _FCRstate = _settings getVariable "fcrInstalled";
+private _FCRstate = [0, 1] select _FCRstate;
+_heli animateSource ["fcr_enable", _FCRstate];
+
+private _IAFSstate = _settings getVariable "iafsInstalled";
+private _IAFSstate = [1, 0] select _IAFSstate;
```

Comment:
For the IAFS and FCR state variables, can we merge the two lines into `[1, 0] select (_settings getVariable "iafsInstalled")